number-guessing-game/.gitignore

```diff
@@ -4,7 +4,6 @@ node_modules
 
 # Hardhat files
 cache
-artifacts
```

Comment:
Typically artifacts aren't stored in the code, is there a reason to store them ?

Reply:
True; Hardcoded the address & added the ABIs because cant be build.

Reply:
we need to fix them so :\ otherwise we're bumping the problem to the future.